types/client.d.ts

```diff
@@ -43,6 +43,8 @@ declare namespace Client {
     maxRequestsPerClient?: number;
     /** Max response body size in bytes, -1 is disabled */
     maxResponseSize?: number | null;
+    /** TCP keepalive interval for the socket in milliseconds. Default: `60e3` milliseconds (60s) */
+    tcpKeepAliveInterval?: number | null;
```

Comment:
```suggestion
    tcpKeepAliveInterval?: number | null;
```

Can you please creating a test for this new type? We use tsd.